CONTRIBUTING.md

```diff
@@ -15,6 +15,9 @@ If you find a bug in the source code or a mistake in the documentation, you can
 submitting an issue to our [GitHub Repository][github_newissue]. Even better you can submit a Pull Request
 with a fix.
 
+**Working on your first Pull Request?** You can learn how from this *free* series
```

Comment:
We will need this on here: https://github.com/karma-runner/karma/blob/master/docs/dev/01-contributing.md as well as that is the front facing page on the website to get started